### PR TITLE
fix self, add workCompleteOnKey

### PR DESCRIPTION
**Pull request by jmccarthy14**
@iproctor

### CODE REVIEW

**Comment by iproctor** [comment]
yup

File: index.js [hunk 1]
```diff
@@ -49,20 +49,33 @@ Sevnup.prototype.loadAllKeys = function loadAllKeys() {
...
+ * When you are done working on a key, or no longer want it within bookkeeping
+ * you can alert sevnup to forget it.  This notifies the ring that it doesn't
+ * need attention in the event this node goes down or hands off ownership.
+ * We want the service to be ignorant of vnodes so we rediscover the vnode.
+ * @param {string} key The key you have finished work on.
+ */
+Sevnup.prototype.workCompleteOnKey = function workCompleteOnKey(key) {
+    var self = this;
+    var vnode = self.getVNodeForKey(key);
+    self.vnodeStore.removeKeyFromVNode(vnode, key);
```
Comment (jwolski) [inline index.js]: what about a callback, at the very least an optional one?

File: index.js [hunk 2]
```diff
@@ -49,20 +49,33 @@ Sevnup.prototype.loadAllKeys = function loadAllKeys() {
 };
 
 /**
+ * When you are done working on a key, or no longer want it within bookkeeping
+ * you can alert sevnup to forget it.  This notifies the ring that it doesn't
+ * need attention in the event this node goes down or hands off ownership.
+ * We want the service to be ignorant of vnodes so we rediscover the vnode.
+ * @param {string} key The key you have finished work on.
+ */
+Sevnup.prototype.workCompleteOnKey = function workCompleteOnKey(key) {
```
Comment (jwolski) [inline index.js]: update README too?

Comment (jmccarthy14) [inline index.js]: yes, README has a couple things pending, this is just to race to keep up with IAIN

File: index.js (hunk 1 above)
Comment (jmccarthy14) [inline index.js]: SUPER against this if you want to fight come talk to me

Comment (jwolski) [inline index.js]: then you are leaving the responsibility of logging/error-handling to `vnodeStore`? that should be detailed/documented somewhere.

Comment (jwolski) [inline index.js]: i dont see vnode_store.js handling errors either. we need some insight into the underlying behavior of sevnup.

**Pull request merged**